<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVoucher</class>
 <widget class="QMainWindow" name="AddVoucher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblMa">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã Voucher</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên Voucher:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLuu">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../DoAnCTDL/res/icon/save.png</normaloff>../../DoAnCTDL/res/icon/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHuy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Huỷ</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../DoAnCTDL/res/icon/discard.png</normaloff>../../DoAnCTDL/res/icon/discard.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditMa">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditTen">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblThem">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>THÊM VOUCHER MỚI</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblGiatri">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>193</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Giá trị Voucher</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thời gian áp dụng từ:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditTimefrom">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTimeden">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>đến:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditTimeto">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTen_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>135</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Áp dụng cho:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxLoaiVoucher">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblphantram">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(%)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspinBoxGiatri">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
